java/api/src/main/java/com/exadel/frs/core/trainservice/controller/VerifyController.java

```diff
@@ -43,7 +43,9 @@
             @ApiParam(value = "The minimal percent confidence that found face is actually a face.")
             @RequestParam(value = "det_prob_threshold", required = false) final Double detProbThreshold,
             @ApiParam(value = "Comma-separated types of face plugins. Empty value - face plugins disabled, returns only bounding boxes")
-            @RequestParam(value = "face_plugins", required = false, defaultValue = "") final String facePlugins
+            @RequestParam(value = "face_plugins", required = false, defaultValue = "") final String facePlugins,
+            @ApiParam(value = "Special parameter to show execution_time and plugin_version fields. Empty value - both fields eliminated")
```

Comment:
why this description only in this endpoint?

Reply:
Description added to other endpoints as well